tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js

```diff
@@ -347,7 +347,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             fluid.each(memberNames, function (memberName) {
                 var comp = parentPanel[memberName];
                 jqUnit.assertNotUndefined("The " + memberName + " should be instantiated", comp);
-                jqUnit.assertDeepEq("The context for " + memberName + " should be retained", comp.origContext, comp.container[0].ownerDocument);
```

Comment:
I ran this test in Justin's branch on my Mac and it now passes with no failure messages in Safari.